climakitae/data/variable_descriptions.csv

```diff
@@ -0,0 +1,22 @@
+name,description,extended_description
+T2,Air Temperature at 2m,Temperature of the air 2 meters above Earth's surface. This is the measure of air temperature used for most modeling applications.
+TOT_PRECIP,Precipitation (total),Total precipitation. Computed by summing total gridscale precipitation and total cumulus precipitation.
+REL_HUMIDITY,Relative Humidity,The percentage of water vapor (i.e. moisture) relative to the amount of water vapor needed to saturate the air at the current air temperature and pressure.
+WIND_MAG,Wind Magnitude at 10m,Wind speed at 10 meters above the Earth's surface. Computed by taking the vector magnitude of the west-east (u) component of wind and the north-south (v) component of wind.
+U10,West-East component of Wind at 10m,"Wind coming from the west, measured at 10 meters above Earth's surface. By covention, wind coming from the east is negative."
+V10,North-South component of Wind at 10m,"Wind coming from the north, measured at 10 meters above Earth's surface. By convention, wind coming from the south is negative."
+PSFC,Surface Pressure,Air pressure at Earth's surface.
+SNOWNC,Snowfall (snow and ice),Total precipitation in the form of snow and ice. 

```

Comment:
For clarity and consistency with `Precipitation (total)`, I'd recommend changing the `Snowfall (snow and ice)` description to "Portion of the total precipitation in the form of snow and ice."

Additionally, the order of snowfall could be moved downwards in the list to reside above RAINC and RAINNC, but that is minor and arguably just an opinion, and could stay as is re: order.

Reply:
Agreed! Thanks for your input! :D